kong/utils.go

```diff
@@ -2,8 +2,18 @@ package kong
 
 import (
 	"bytes"
+	"fmt"
 	"net/http"
+	"regexp"
 	"strings"
+
+	"github.com/blang/semver/v4"
+)
+
+var (
+	versionParts   = 4
+	kong140Version = semver.MustParse("1.4.0")
+	kong232Version = semver.MustParse("2.3.2-0")
```

Comment:
I had to use 2.3.2-0 instead of 2.3.2 because 2.3.2-0 is considered as a pre build of 2.3.2